src/app/welcome-page/welcome-page.component.html

```diff
@@ -53,9 +53,9 @@ <h1>
       <h2><span> Document the evidence,</span> unmask the trolls</h2>
 
       <p>
-        Use FeedShield to flag abusive content targeting you, so you can report the
-        abuse to the platforms or get our forensic investigators to track the
-        culprits.
+        Find out how to use FeedShield to flag abusive content targeting you, so you report the abuse to the platforms
+        or get
+        our forensic investigators to track the culprits.
```

Comment:
This doesn't read right. Feels like a "can" is missing somewhere. Ping me on the source.